lib/containers/DiscoveredDevices.jsx

```diff
@@ -98,6 +99,20 @@ class DiscoveredDevices extends React.PureComponent {
         toggleOptionsExpanded();
     }
 
+    handleTimoutChange(value) {
```

Comment:
Timout ;)

Also shouldn't we call bind on this in the constructor like we do for handleActiveScanCheck? Otherwise this.props will be undefined?

Reply:
I'm not 100% sure, but since I render it with an arrow function it will be bounded.
But i guess that can be changed such that all onChange events are alike

Reply:
I'm not 100% sure, but since I render it with an arrow function it will be bounded.
But i guess that can be changed such that all onChange events are alike

Reply:
Yeah tested it now and it seems you are right. No need to change it further :)